### PR TITLE
Move to google blockly

### PR DESCRIPTION
**Pull request by AndriyPt**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,18 +43,26 @@ cd ../
 catkin_make_isolated --pkg robot_blockly --install
 ```
 
-To start the backend and build in HTTP server:
+To start the backend and build-in HTTP server:
```
Comment (kirstyellis) [inline README.md]: built-in

File: README.md [hunk 2]
```diff
@@ -43,18 +43,26 @@ cd ../
...
 ```
 source install_isolated/setup.bash 
 roslaunch robot_blockly robot_blockly.launch 
 ```
 
+In order to include blocks library (e.g. [sr_blockly_blocks](https://github.com/shadow-robot/sr_blockly_blocks)) run:
+```
+source install_isolated/setup.bash
```
Comment (kirstyellis) [inline README.md]: install_isolated, should be devel?

File: README.md [hunk 3]
```diff
@@ -43,18 +43,26 @@ cd ../
 catkin_make_isolated --pkg robot_blockly --install
 ```
 
-To start the backend and build in HTTP server:
+To start the backend and build-in HTTP server:
 ```
 source install_isolated/setup.bash
```
Comment (kirstyellis) [inline README.md]: seems to be a duplicate of the lines below

File: README.md [hunk 4]
```diff
@@ -43,18 +43,26 @@ cd ../
...
+roslaunch robot_blockly robot_blockly.launch block_packages:=[sr_blockly_blocks]
+```
+
 Point your browser to: [http://localhost:8000/pages/blockly.html](http://localhost:8000/pages/blockly.html)
 
 ### Create your own blocks
-- Open [http://localhost:8000/blockly/demos/blockfactory/index.html](http://localhost:8000/blockly/demos/blockfactory/index.html)
-- Design you own block and then add the metadata to: `frontend/blockly/blocks` and `frontend/blockly/generator`
-- Launch `python build.py` to regenerate the blocks.
+- Default blockly blocks are stored in the blockly package. If you require additional block repositories for your project, these repositories should be cloned into your workspace at the same level as robot_blockly. Then rebuild your workspace.
```
Comment (kirstyellis) [inline README.md]: so, here you will need to say that the sr_blockly package is required for the default shadow blocks and tools, then the additional block repos can be added/cloned into the workspace

File: README.md [hunk 5]
```diff
@@ -43,18 +43,26 @@ cd ../
...
+- If you want to create your own blocks, you can either add them to an existing ros package or create a new package.
+- Blocks can be generated by opening [http://localhost:8000/blockly/demos/blockfactory/index.html](http://localhost:8000/blockly/demos/blockfactory/index.html)
```
Comment (kirstyellis) [inline README.md:64]: should this be:  [http://localhost:8000/blockly/demos/blockfactory/index.html](frontend/blockly/demos/blockfactory/index.html)?

File: README.md [hunk 6]
```diff
@@ -43,18 +43,26 @@ cd ../
...
+- Design you own block and then add the metadata to: `blocks` and `generators` in the block repository. An xml snippet should be included in any Javascript file of the block repository, please see [sr_blockly_blocks](https://github.com/shadow-robot/sr_blockly_blocks/toolbox) for an example of how to do this.
```
Comment (kirstyellis) [inline README.md]: should be sr_blockly now

File: README.md [hunk 7]
```diff
@@ -43,18 +43,26 @@ cd ../
...
+- catkin_make will automatically merge all Javascript files into single file. So in case if you have changed Javascript file you need to run catkin_make.
```
Comment (kirstyellis) [inline README.md]: "into a single file"

File: README.md (hunk 5 above)
Comment (AndriyPt) [inline README.md:64]: Did you mean just URL text without all these [] and () ?

File: scripts/pywebserver.py [hunk 8]
```diff
@@ -4,20 +4,52 @@
...
+import rospy
+
+rospy.init_node("robot_blockly_webserver")
 
 rp = RosPack()
 frontend_path = rp.get_path('robot_blockly')
 frontend_path += '/frontend'
 
-print("Changing serve path to: " + frontend_path)
+rospy.loginfo("Changing serve path to: " + frontend_path)
```
Comment (kirstyellis) [inline scripts/pywebserver.py]: server path?

**Pull request merged**